impl/otbn/mont_loop.i.vad

```diff
@@ -322,19 +322,29 @@ procedure mont_loop_1(
             next_y_it.buff, next_m_it.buff, prev_view, next_view, j+1);
 {
     ghost var j := y_it.index;
-    /* load limb of y (operand a) and mult. with xi: [w26, w27] <- y[j]*xi */
+    comment("/* Load limb of y (operand a): w30 <= y[j] */");
```

Comment:
the `<=` might be misunderstood as less than equal, might want to consider using `=` or `==`.